susemanager-utils/susemanager-sls/susemanager-sls.spec

```diff
@@ -91,6 +91,7 @@ cp src/modules/mgrclusters.py %{buildroot}/usr/share/susemanager/salt/_modules
 cp src/modules/mgr_caasp_manager.py %{buildroot}/usr/share/susemanager/salt/_modules
 cp src/modules/ssh_agent.py %{buildroot}/usr/share/susemanager/salt/_modules
 cp src/states/product.py %{buildroot}/usr/share/susemanager/salt/_states
+cp src/states/mgrcompat.py %{buildroot}/usr/share/susemanager/salt/_states
```

Comment:
I think using `module.run` is now "forbidden". Would it make sense to do a simple test to check that nobody uses it when pushing changes in future?

Maybe simply a 

```
grep "module.run" %{buildroot}/usr/share/susemanager/salt || exit 1
```

in %check section or something similar in the unit tests which can run also on PRs.

Reply:
Yeah, we should forbid the usage of `module.run` in our SLS files from now on. So, those RPM checks or unit tests you propose sound perfect.

I'll push also those checks into this PR.